recipes/netpbm/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set version = "10.68" %}
+
+package:
+  name: netpbm
+  version: {{ version }}
+
+source:
+  # See http://netpbm.svn.sourceforge.net/viewvc/netpbm/
+  svn_url: http://svn.code.sf.net/p/netpbm/code/advanced
+  svn_rev: 2294
```

Comment:
The rationale as stated in my last comment mentioned that this is an issue of stability not svn.

Could you please explain what features you need that are not included in 10.47.60?